src/app/configurator/configurator.component.ts

```diff
@@ -1,20 +1,73 @@
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { ConfiguratorService } from '../api/configurator.service';
+import { IProduct } from '../Interface/products';
+import { FormBuilder, FormControl, Validators } from '@angular/forms';
+import { FormGroup } from '@angular/forms';
 
 
 @Component({
   selector: 'app-configurator',
   templateUrl: './configurator.component.html',
   styleUrls: ['./configurator.component.scss']
 })
+
 export class ConfiguratorComponent implements OnInit {
 
+  configurator: string | undefined;
```

Comment:
Seems like this is the action (cta). If so please rename the variable to make it recognisable

Reply:
done